src/display.rs

```diff
@@ -188,6 +188,25 @@ where
         Ok(())
     }
 
+    /// Send the full framebuffer to the display
+    ///
+    /// This resets the draw area the full size of the display
+    #[cfg(feature = "async")]
+    pub async fn flush_async(&mut self) -> Result<(), Error<CommE, PinE>> {
+        use embassy_traits::spi::Write as SpiTrait;
+
+        // Ensure the display buffer is at the origin of the display before we send the full frame
+        // to prevent accidental offsets
+        self.set_draw_area((0, 0), (DISPLAY_WIDTH - 1, DISPLAY_HEIGHT - 1))?;
+
+        // 1 = data, 0 = command
+        self.dc.set_high().map_err(Error::Pin)?;
+
+        self.spi.write(&self.buffer).map_err(Error::Comm)?;
```

Comment:
Nothing is awaited here, which means I don't think this function is actually async.

Reply:
Yeah its weird. clippy got pissy when I had it return a future, it seems to like it this way
I use it here
https://github.com/jacobrosenthal/nrf52840-mdk-rs/blob/amg88/src/display.rs#L107

Reply:
I think you're right @derekdreery the bound on 
```
where
    SPI: hal::blocking::spi::Write<u8, Error = CommE>,
```
Is still locking me to that blocking trait